### DISCUSSION TITLE
limit 1000

### DISCUSSION THREAD

**Discussion by cat228608** [xtekky/gpt4free#2232] [Q&A]
MSG: One message exceeds the 1000chars per message limit. Join our discord for more: [https://discord.com/invite/q55gsH8z5F](https://discord.com/invite/q55gsH8z5F)

Code:
```
def chatgpt(query):
    print(f"[LOG] GPT compiling...")
    try:
        client = Client()
        response = client.chat.completions.create(
            model="gpt-4",
            messages=[{"role": "user", "content": query}],
        )
        return response.choices[0].message.content
    except Exception as e:
        print("[ERROR gpt] " + e)
```

**Comment by Parthsadaria** [comment]
dont use airforce provider that leads to the issue 
```client = Client(g4f.provider.{providername})```
use some other provider at place of providername
